### PR TITLE
Paradise Lost and It's Consequences Have Been Disastrous for Lobotomy Corp 13 endgame.

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
### -------THIS IS A SPITE PR.-------
Paradise Lost nerfed to 7/7/7/7
Realization can be worn at 100s, still requires 130 stats to realize, and still subtracts 30 from all stats.

## Why It's Good For The Game
It's an abno that's easy in containment that is a bitch when it breaches and it gives you armor that invalidates EVERY SINGLE ALEPH ARMOR Except for 3.
It's not even significantly harder than most alephs either, just slightly more annoying when it breaches, and super tame in containment.
There is NO reason that it should make every armor entirely useless except for Smile, Mimicry and Da Capo. 

Honestly, IMO, the very existence of PL makes endgame significantly easier. This will make endgame a lot harder because you don't get 80% armor in everything, and now have to grind and choose armor for the circumstances.

Realized armor can now be worn at all 100s. You no longer have to hold on to armor for a million years while you grind it up.
You are still weaker though, and it still requires all 130s, technically.

## Changelog
:cl:
balance: rebalanced endgame armor.
/:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
Requires Egor's Eye.

**Comment by SpaceJohn413** [comment]
based

File: code/modules/clothing/suits/ego_gear/aleph.dm [hunk 1]
```diff
@@ -1,12 +1,11 @@
-// ALEPH armor, go wild, but attempt to keep total armor at ~245 total.
-// Paradise lost and Twilight are the only exceptions.
```
Comment (EgorDinamit) [inline code/modules/clothing/suits/ego_gear/aleph.dm:2]: Read this.

File: code/modules/clothing/suits/ego_gear/realized.dm [hunk 2]
```diff
@@ -4,10 +4,10 @@
...
 	worn_icon = 'icons/mob/clothing/ego_gear/realized.dmi'
 	attribute_requirements = list(
-							FORTITUDE_ATTRIBUTE = 130,
-							PRUDENCE_ATTRIBUTE = 130,
-							TEMPERANCE_ATTRIBUTE = 130,
-							JUSTICE_ATTRIBUTE = 130
+							FORTITUDE_ATTRIBUTE = 100,
+							PRUDENCE_ATTRIBUTE = 100,
+							TEMPERANCE_ATTRIBUTE = 100,
+							JUSTICE_ATTRIBUTE = 100
```
Comment (EgorDinamit) [inline code/modules/clothing/suits/ego_gear/realized.dm:10]: The only change I approve of.

File: code/modules/clothing/suits/ego_gear/aleph.dm [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
+// ALEPH armor, go wild, but attempt to keep total armor at ~240 total.
 
 /obj/item/clothing/suit/armor/ego_gear/paradise
 	name = "paradise lost"
 	desc = "\"My loved ones, do not worry; I have heard your prayers. \
 	Have you not yet realized that pain is but a speck to a determined mind?\""
 	icon_state = "paradise"
-	armor = list(RED_DAMAGE = 80, WHITE_DAMAGE = 80, BLACK_DAMAGE = 80, PALE_DAMAGE = 80) // 320
+	armor = list(RED_DAMAGE = 60, WHITE_DAMAGE = 60, BLACK_DAMAGE = 60, PALE_DAMAGE = 90) // 270
```
Comment (EgorDinamit) [inline code/modules/clothing/suits/ego_gear/aleph.dm:8]: No. PL stays the same.

File: code/modules/clothing/suits/ego_gear/aleph.dm [hunk 4]
```diff
@@ -69,7 +68,7 @@
 	desc = "The three birds united their efforts to defeat the beast. \
 	This could stop countless incidents, but you’ll have to be prepared to step into the Black Forest…"
 	icon_state = "twilight"
-	armor = list(RED_DAMAGE = 90, WHITE_DAMAGE = 90, BLACK_DAMAGE = 90, PALE_DAMAGE = 90) // 360
+	armor = list(RED_DAMAGE = 80, WHITE_DAMAGE = 80, BLACK_DAMAGE = 80, PALE_DAMAGE = 50) // 290
```
Comment (EgorDinamit) [inline code/modules/clothing/suits/ego_gear/aleph.dm:71]: It is planned to give twilight armor an ability(like in LC), so there's no need to change it until it's done.

File: code/modules/clothing/suits/ego_gear/aleph.dm (hunk 1 above)
Comment (Kitsunemitsu) [reply code/modules/clothing/suits/ego_gear/aleph.dm:2]: They're still better than all the other aleph armor.

File: code/modules/clothing/suits/ego_gear/aleph.dm (hunk 4 above)
Comment (Kitsunemitsu) [reply code/modules/clothing/suits/ego_gear/aleph.dm:71]: I can revert this one, but Paradise lost is NOT okay.

File: code/modules/clothing/suits/ego_gear/aleph.dm (hunk 3 above)
Comment (Kitsunemitsu) [reply code/modules/clothing/suits/ego_gear/aleph.dm:8]: It's too fucking strong egor. It was fine back when every time WN breached 12 people died. Now they just keep it in containment, which, is not hard.

File: code/modules/clothing/suits/ego_gear/aleph.dm [hunk 5]
```diff
@@ -69,7 +68,7 @@
 	desc = "The three birds united their efforts to defeat the beast. \
 	This could stop countless incidents, but you’ll have to be prepared to step into the Black Forest…"
 	icon_state = "twilight"
-	armor = list(RED_DAMAGE = 90, WHITE_DAMAGE = 90, BLACK_DAMAGE = 90, PALE_DAMAGE = 90) // 360
+	armor = list(RED_DAMAGE = 80, WHITE_DAMAGE = 90, BLACK_DAMAGE = 90, PALE_DAMAGE = 90) // 290
```
Comment (EgorDinamit) [inline code/modules/clothing/suits/ego_gear/aleph.dm:71]: ?

Comment (Kitsunemitsu) [reply code/modules/clothing/suits/ego_gear/aleph.dm:71]: I forgot to change an 80 back to a 90.

File: code/modules/clothing/suits/ego_gear/aleph.dm [hunk 6]
```diff
@@ -1,12 +1,11 @@
...
-// Paradise lost and Twilight are the only exceptions.
+// ALEPH armor, go wild, but attempt to keep total armor at ~240 total.
 
 /obj/item/clothing/suit/armor/ego_gear/paradise
 	name = "paradise lost"
 	desc = "\"My loved ones, do not worry; I have heard your prayers. \
 	Have you not yet realized that pain is but a speck to a determined mind?\""
 	icon_state = "paradise"
-	armor = list(RED_DAMAGE = 80, WHITE_DAMAGE = 80, BLACK_DAMAGE = 80, PALE_DAMAGE = 80) // 320
+	armor = list(RED_DAMAGE = 70, WHITE_DAMAGE = 70, BLACK_DAMAGE = 70, PALE_DAMAGE = 70) // 270
```
[the remark this answers is not in the document]
Comment (Kitsunemitsu) [reply code/modules/clothing/suits/ego_gear/aleph.dm:8]: You said I specifically could do this.

[the remark this answers is not in the document]
Comment (EgorDinamit) [reply code/modules/clothing/suits/ego_gear/aleph.dm:8]: Yeah, just saying the same thing here.

[the remark this answers is not in the document]
Comment (EgorDinamit) [reply code/modules/clothing/suits/ego_gear/aleph.dm:8]: Also it's 280 now, not 270.

[the remark this answers is not in the document]
Comment (EgorDinamit) [reply code/modules/clothing/suits/ego_gear/aleph.dm:8]: Same thing with apocalypse bird EGO. Comments do matter.

**Review by EgorDinamit — CHANGES_REQUESTED** [review]
Change cost of Paradise Lost to 200 and then squash commits.

**Review by EgorDinamit — APPROVED** [review]

**Pull request merged**